### ISSUE TITLE
Samplesheet description in the documentation

### ISSUE THREAD

**Issue by IStevant** [nf-core/chipseq#320]
### Description of the bug

Dear developers,

I noticed the example of samplesheet is not correct, especially the `control` column.
The `sample` column contains the name of the samples plus the replicate number, while in the `control` column the sample name do not contain the replicate number information.

I was trying to adapt the example with my samples and I had the following error message:

```
sample,fastq_1,fastq_2,antibody,control
E11_5_IP_REP1,NWJS1.R1.fastq.gz,NWJS1.R2.fastq.gz,myc,Input
E11_5_IP_REP2,NWJS2.R1.fastq.gz,NWJS2.R2.fastq.gz,myc,Input
E11_5_contr_IP_REP1,NWJS3.R1.fastq.gz,NWJS3.R2.fastq.gz,myc,Input
E11_5_contr_IP_REP2,NWJS4.R1.fastq.gz,NWJS4.R2.fastq.gz,myc,Input
Input_REP1,NWJS13.R1.fastq.gz,NWJS13.R2.fastq.gz,,
Input_REP2,NWJS14.R1.fastq.gz,NWJS14.R2.fastq.gz,,
```

```
Command wrapper:
  ERROR: Please check samplesheet -> Control identifier has to match does a provided sample identifier!
  Control: 'Input'
```

I had to add the '_REP1 or 2' to make the pipeline run.

Could you correct the examples in the documentation?

Best,

I.

**Comment by sanjeevRJMU1** [comment]
@IStevant thank you so much for pointing out this issue and your fix. I ran into the same issue (so frustrating!) and was able to resolve this by adding the "REPn" nomenclature to the inputs as you mentioned. :)

**Comment by KrebsLab** [comment]
I'm also facing the same issue. Splitting the sample column to sample and replicate would be neat

**Comment by JoseEspinosa** [comment]
> I'm also facing the same issue. Splitting the sample column to sample and replicate would be neat

Thanks for the comments. This is already implemented in #349 and should be shortly released
